myapp/config/locales/common.ja.yml

```diff
@@ -0,0 +1,8 @@
+ja:
+  ops:
+    edit: '編集'
+    delete: '削除'
+  page_titles:
+    top_page: 'トップページ'
+  messages:
+    confirm_delete: '本当に削除しますか？'
```

Comment:
Please give me a line break at the end!
I have my editor set to put a new line at the end, I recommend it!

Reply:
Thank you I fixed this